ci_framework/roles/edpm_build_images/defaults/main.yml

```diff
@@ -0,0 +1,37 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_edpm_build_images"
+
+cifmw_edpm_build_images_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework') }}"
+cifmw_edpm_build_images_src: "https://github.com/openstack-k8s-operators/edpm-image-builder"
+cifmw_edpm_build_images_artifacts: "{{ cifmw_edpm_build_images_basedir }}/artifacts/edpm_images"
+cifmw_edpm_build_images_packages:
+  - python3-devel
+  - python3-pip
+  - podman
+  - buildah
+  - qemu-img
```

Comment:
have a look at diskimage-builder/bindep.txt for a full list of package requirements, there is a few missing. The lack of dosfstools is causing the specific failure in [1]

[1] https://logserver.rdoproject.org/76/176/fa046db96982bf604b4d14c1cf1e7c5258a59541/github-check/cifmw-edpm-build-image/4b6834b/ci-framework/artifacts/edpm_images/edpm_hardened_uefi_image_build.log